content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,121 @@
+<input type="hidden" name="language" value="de">
+<input type="hidden" name="year" value="2021">
+<div style="float: right; margin-top: 10px;"><input class="button button-primary" type="submit" value="Registrieren"></label></div>
+<div class="clearfix"></div>
+</form>
+</div>
+
+Du hast aber auch die Möglichkeit, ohne Nutzung von GitHub teilzunehmen. Schick uns dazu bitte eine E-Mail an <hacktoberfest@datenanfragen.de>. In der E-Mail muss der Wunsch, an der Aktion teilzunehmen, das Einverständnis mit unseren [Teilnahme- und Datenschutzbedingungen](https://static.dacdn.de/docs/bedingungen-hacktoberfest-2021.pdf) und mindestens ein Git-Patch enthalten sein.
+
+Übrigens: Wenn Du _zusätzlich_ beim offiziellen Hacktoberfest teilnehmen möchtest, musst Du Dich dafür [separat registrieren](https://hacktoberfest.digitalocean.com/). Das ist aber **nicht** nötig, um nur an unserer Aktion teilzunehmen.
+
+
+<a id="repos"></a>
+
+## Welche Repositories zählen für die Aktion?
+
+Für die Aktion werten wir nur Pull-Requests und Patches in den folgenden Repositories als gültige Beiträge:
+
+* <https://github.com/datenanfragen/website>
+* <https://github.com/datenanfragen/data>
+* <https://github.com/datenanfragen/backend>
+* <https://github.com/datenanfragen/company-json-generator>
+* <https://github.com/datenanfragen/letter-generator>
+* <https://github.com/datenanfragen/media>
+* <https://github.com/datenanfragen/locate-contacts-addon>
+* <https://github.com/zner0L/postcss-fonticons>
+
+In Ausnahmefällen können wir auch weitere Repositories erlauben. Wenn Du meinst, dass Dein Beitrag in einem anderen Repository dem Projekt hilft, frag bitte vorher kurz per E-Mail an <hacktoberfest@datenanfragen.de> nach, ob wir diesen Beitrag auch werten können.
+
+## Habt Ihr Vorschläge für Beiträge?  
+
+<img class="offset-image offset-image-right" src="/card-icons/code.svg" height="150px" width="190px" style="height: 150px; margin-right: -100px; margin-top: -50px;" alt="">
+
+Gerade wenn Du neu bei Datenanfragen.de bist, weißt Du wahrscheinlich gar nicht, wo Du anfangen kannst. Keine Sorge: Es gibt bei uns zahlreiche Bereiche, in denen Du mitmachen kannst – ganz unabhängig davon, ob Du Erfahrung mit Programmierung und Datenschutz hast oder nicht. Zum leichteren Einstieg haben wir Dir hier einmal ein paar Vorschläge zusammengestellt. Viele weitere Aufgaben findest Du in den Issues der jeweiligen Repositories. Wir freuen uns natürlich auch, wenn Du eigenen Ideen hast, was Du am Projekt verbessern könntest.
+
+* **Zusammentragen von empfohlenen Unternehmen für weitere Länder**  
```

Comment:
It's reasonably unlikely that a person reading the German post would be able to help with this (or the translation tasks). We should at the very least reorder the list to make more sense in this regard. I would prefer if we found some more tasks that can be done without knowing languages other than German and English.